WRFotron2.0_CEMAC_ARC4/namelist.wrf.blueprint

```diff
@@ -43,8 +38,8 @@
  e_we                                = 140, 154,                             ! westeast dimension
  e_sn                                = 140, 154,                             ! southnorth dimension
  e_vert                              = 33, 33,                               ! vertical dimension
- num_metgrid_levels                  = 38,                                   ! number of vertical levels in WPS output
- num_metgrid_soil_levels             = 3,                                    ! number of soil levels or layers in WPS output
+ num_metgrid_levels                  = 27,                                   ! number of vertical levels in WPS output
```

Comment:
ECMWF requires `num_metgrid_levels = 38 `. I think the CEMAC compilation uses this right?

Reply:
I've just copied over the WRFototron2.0 namelist files and replaced. My tests were using GFS data, do you want me to default to ecmwf? I've swapped in config.bash and pre.bash to default to ecmwf.

Reply:
Okay thanks. 
Most do use ECMWF now, so that's useful.